gitpod-io/website
# Fix docs for deprection of projects under personal accounts.

Adjust docs to reflect deprecation of projects under personal accounts (see https://github.com/gitpod-io/gitpod/issues/14035).

Also removes mention of project configuration pages, and removes the link to the GitHub marketplace.  

Fixes #2934

#### How to test

See preview pages for [Projects](https://deploy-preview-2960--gitpod-kumquat.netlify.app/docs/configure/projects) and for [Teams](https://deploy-preview-2960--gitpod-kumquat.netlify.app/docs/configure/teams).


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2960"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2960"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
